src/components/changePassword/index.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component, Fragment } from 'react';
+import { isEmpty } from 'lodash';
+import styles from './styles.module.scss';
+import validateAuth from '../../validation/auth';
+import { changePassword } from '../../services/auth';
```

Comment:
Change imports order: main react libraries - custom react libraries - your components - styles. Each group separate by whitespace. Check it in all your files

Reply:
Maybe it useful to add a rule into eslint

Reply:
Fixed